ast/module.c2

```diff
@@ -70,7 +70,8 @@ public fn void Module.free(Module* m) {
 public fn void Module.setUsed(Module* m) { m.is_used = true; }
 public fn bool Module.isUsed(const Module* m) { return m.is_used; }
 
-public fn void Module.setInternal(Module* m) { m.is_internal = true; }
+// TODO use use a keyword or an attribute for this
```

Comment:
Internal means it's in the user code and not a library. No keyword/attribute is needed since it's a compiler internal thing